### ISSUE TITLE
Custom pre-configured test groups for specific pentests/scans

### ISSUE THREAD

**Issue by anantshri** [owtf/owtf#331]
Right now we have two tabs 
individual 
groups.

lets introduce another tab which could be preconfigured settups like

1) OWASP top 10 web 2013
2) web application + networkscan.
3) Linux server scan
4) Windows server scan
5) Voip scan
and so on i suppose you got the idea.

for all these there will be a hybrid group mapping required.

**Comment by delta24** [comment]
@saganshul Can you find a way for a user to be able check some plugins, save them to a new custom group and run it? This will all be client-side (**for the moment**) and the plugins will execute similar to how we execute multiple individual plugins.

**Comment by saganshul** [comment]
@delta24, yeah I think we can do that using local memory storage.

**Comment by delta24** [comment]
@saganshul Yes, for the moment - in the near future, we need to save the groups on the DB too. This would be like the `Zest` thing - if a user selects more than one plugin to run, then a option should be available to save them in a custom group (with user specified name) for future use.

**Comment by avinxshKD** [comment]
@viyatb can i work on this if its available?
